src/pages/help-centre/components/_questions-section.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import styled from 'styled-components'
 import { TQuestionsData } from '../data/_data-types'
-import { useFilteredCategory } from '../data/_hooks'
+import { useFilteredCategory, useFilteredP2PCategory } from '../data/_hooks'
```

Comment:
I think we don't need to create new filter function, we can just add the check in the same `useFilteredCategory` function.